src/genconfig.sh

```diff
@@ -23,6 +23,28 @@ if [ ! -z "$1" ]; then
 	KERNEL_VERSION="$1"
 fi
 
+# As a fallback mechanism, if System.map is not found, download
+# the debug linux kernel package and extract it from there
+extract_system_map() {
```

Comment:
May be rename extract_system_map_for_deb?

Reply:
Done.